test/src/misc/getHighlightClassName.js

```diff
@@ -14,13 +14,6 @@ test("Return the highlight classname if a date should be highlighted", function(
 	strictEqual(cal.getHighlightClassName(new Date(2000, 0, 1)), " highlight");
 });
 
-test("Return the highlight and now classname if a date should be highlighted and is now", function() {
-	expect(1);
-
-	var cal = createCalendar({highlight: [new Date(2000, 0, 1), new Date()]});
-	strictEqual(cal.getHighlightClassName(new Date()), " highlight now");
```

Comment:
Replacing `highlight now` by `highlight highlight-now` should make the test pass